.github/actions/check-size/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Check size of a policy package
+description: Checks the disk size of a policy package
+
+inputs:
+  package:
+    description: the name of the package
+    required: true
+  limit:
```

Comment:
Can we make this a `type: integer` and pass an integer and not a string to the action?

Reply:
See earlier answer to same. I guess, sure, but .... such slow runs.

Reply:
Done!